test/e2e/propagationpolicy_test.go

```diff
@@ -513,9 +491,67 @@ var _ = ginkgo.Describe("[BasicPropagation] propagation testing", func() {
 	})
 })
 
+var _ = ginkgo.Describe("[CornerCase] PropagationPolicy testing", func() {
+	var policyNamespace, policyName string
+	var policy *policyv1alpha1.PropagationPolicy
+
+	ginkgo.JustBeforeEach(func() {
+		framework.CreatePropagationPolicy(karmadaClient, policy)
+		ginkgo.DeferCleanup(func() {
+			framework.RemovePropagationPolicy(karmadaClient, policy.Namespace, policy.Name)
+		})
+	})
+
+	ginkgo.Context("Propagate Deployment with long pp name (exceed 63 character)", func() {
+		var deploymentNamespace, deploymentName string
+		var deployment *appsv1.Deployment
+
+		ginkgo.BeforeEach(func() {
+			policyNamespace = testNamespace
+			policyName = deploymentNamePrefix + "-longname-longname-longname-longname-longname-longname-" + rand.String(RandomStrLength)
```

Comment:
I think need to  verify `policyName` length > 63

Reply:
Yes, the length of the policy name exceeds 63 characters.